solanaj/src/main/java/org/p2p/solanaj/utils/SolanjLogger.kt

```diff
@@ -0,0 +1,19 @@
+package org.p2p.solanaj.utils
+
+/**
+ * Can't use Timber in this module, but sometimes logging is needed
+ * so we've created an interface and have it's implementation in main :app
+ */
+interface SolanjLogger {
```

Comment:
we can't log with Timber in Solanj module directly, so I created a interface to allow us to do this